### PR TITLE
Addition of repo widget and syntax changes to make more general 

### PR DESCRIPTION
**Pull request by Courtneyp123**
Repo widget edits and design changes.  

Most updated code that minimizes code.

### CODE REVIEW

File: src/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (Sbrenthughes) [inline src/.vscode/launch.json]: This file needs to be removed and added to the git ignore

Comment (Courtneyp123) [reply src/.vscode/launch.json]: Screen Shot 2020-03-24 at 10 56 45 AM
Hm.. I thought it was added already.  Would this be a different pathway?

**Comment by rvema** [comment]
@Courtneyp123 can you also fix the lgtm alerts?

**Comment by Courtneyp123** [comment]
lgtm alerts noted.

**Pull request merged**